src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/bothfiles/package-info.java

```diff
@@ -13,4 +13,16 @@
 
 */
 
-package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle.bothfiles; // violation
+/** Example */ // violation
```

Comment:
please add violation message.

we are changing input file by structure (extra comment).

Reply:
Done.